presto-product-tests/src/assembly/scripts.xml

```diff
@@ -0,0 +1,28 @@
+<assembly>
+<id>scripts</id>
+<formats>
+    <format>zip</format>
+</formats>
+
+<fileSets>
+   <fileSet>
+        <directory>./</directory>
+        <includes>
+            <include>README.md</include>
+        </includes>
+    </fileSet>
+    <fileSet>
+        <directory>bin</directory>
+    </fileSet>
+    <fileSet>
+        <directory>conf/docker</directory>
+    </fileSet>
+    <fileSet>
+        <directory>conf/presto</directory>
+    </fileSet>
+    <fileSet>
+        <directory>conf/tempto</directory>
+    </fileSet>
```

Comment:
I thought about that, but you also need the CLI, JDBC driver, and server tar ball if you're really going to be able to run the tests, at which point the zip is huge.  I was also concerned about having duplicate copies of the binaries in our archives.  Seems like we have disk space issues now, and I did not want to make them worse.